setup.py

```diff
@@ -49,6 +49,10 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Topic:: Scientific / Engineering',
+        'Topic:: Scientific / Engineering:: GIS',
```

Comment:
Let's replace `GIS` with one of these keywords: FAIR, BMI, Geoscience. I like Geoscience more.

Reply:
Classifiers have to be picked from https://pypi.org/classifiers/, but I can your suggestions to keywords.